src/Control/Email/Email.php

```diff
@@ -49,9 +49,10 @@ class Email extends ViewableData
 
     /**
      * This will be set in the config on a site-by-site basis
+     * @see https://docs.silverstripe.org/en/4/developer_guides/email/#administrator-emails
      *
      * @config
-     * @var string The default administrator email address.
+     * @var string|array The default administrator email address or array of [email => name]
```

Comment:
The string value is just an email address, do you think the array value should be a list of email addresses? I find it a little odd that we're changing array values to accept email and names

Reply:
Yeah, I agree that API is utterly bad, but that's what Swiftmailer has and what we pass there "as is" and what we've got in the documentation - https://docs.silverstripe.org/en/4/developer_guides/email/#administrator-emails

Reply:
I reckon we should fix that in the next major release though